File: backend/src/main/java/turip/regionCategory/controller/dto/response/RegionCategoriesResponse.java
package turip.regionCategory.controller.dto.response;

import java.util.List;

public record RegionCategoriesResponse(
        List<RegionCategoryResponse> regionCategories
) {

    public static RegionCategoriesResponse from(List<RegionCategoryResponse> regionCategories) {
        return new RegionCategoriesResponse(regionCategories);
    }
} 

File: android/app/src/main/java/com/on/turip/data/region/dto/RegionCategoriesResponse.kt
package com.on.turip.data.region.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class RegionCategoriesResponse(
    @SerialName("regionCategories")
    val regionCategories: List<RegionCategoryResponse>,
)
